pywinauto/unittests/test_uiawrapper.py

```diff
@@ -490,6 +490,13 @@ def test_pretty_print(self):
             assert_regex(wrp.element_info.__repr__(), "^<uia_element_info\.UIAElementInfo - 'None', Window, [0-9-]+>$")
             wrp.element_info._get_name = orig
 
+        def test_pretty_print_encode_error(self):
+            """Test __repr__ method for BaseWrapper with specific Unicode text (issue #594)"""
+            wrp = self.dlg.wrapper_object()
+            wrp.window_text = mock.Mock(return_value=u'\xb7')
+            print(wrp)
+            print(repr(wrp))
```

Comment:
Any special reason do not use assert with a   call to __str__ method of the object?

Reply:
`print(wrp)` calls `str(wrp)` in Python 2.7. Just thought to be close to real-world conditions.